index.js

```diff
@@ -19,25 +19,9 @@ module.exports = {
     /* eslint-enable no-unused-expressions */
   },
 
-  _findHost: function () {
```

Comment:
I know this code is specifically added to support using this addon in ember-engine. Can you confirm that the import still works with the latest engine?

Reply:
@quincyle - I did not test this with an ember-engine. I did however, test this change with an app and a nested add-on.

Please let me know if this does not work with your ember-engine.